ocropus-gpageseg

```diff
@@ -25,66 +25,74 @@ from ocrolib import psegutils,morph,sl
-parser.add_argument('--maxlines',type=float,default=300,
+group_error_checking.add_argument('--maxlines',type=float,default=300,
                     help='maximum # lines permitted, default: %(default)s')
 
 # scale parameters
-parser.add_argument('--scale',type=float,default=0.0,
+group_scale = parser.add_argument_group('scale parameters')
+group_scale.add_argument('--scale',type=float,default=0.0,
                     help='the basic scale of the document (roughly, xheight) 0=automatic, default: %(default)s')
-parser.add_argument('--hscale',type=float,default=1.0,
+group_scale.add_argument('--hscale',type=float,default=1.0,
                     help='non-standard scaling of horizontal parameters, default: %(default)s')
-parser.add_argument('--vscale',type=float,default=1.0,
+group_scale.add_argument('--vscale',type=float,default=1.0,
                     help='non-standard scaling of vertical parameters, default: %(default)s')
 
 # line parameters
-parser.add_argument('--threshold',type=float,default=0.2,
+group_line = parser.add_argument_group('line parameters')
+group_line.add_argument('--threshold',type=float,default=0.2,
                     help='baseline threshold, default: %(default)s')
-parser.add_argument('--noise',type=int,default=8,
+group_line.add_argument('--noise',type=int,default=8,
                     help="noise threshold for removing small components from lines, default: %(default)s")
-parser.add_argument('--usegauss',action='store_true',
+group_line.add_argument('--usegauss',action='store_true',
                     help='use gaussian instead of uniform, default: %(default)s')
 
 # column parameters
-parser.add_argument('--maxseps',type=int,default=2,
+group_column = parser.add_argument_group('column parameters')
+group_column.add_argument('--maxseps',type=int,default=2,
                     help='maximum black column separators, default: %(default)s')
-parser.add_argument('--sepwiden',type=int,default=10,
+group_column.add_argument('--sepwiden',type=int,default=10,
                     help='widen black separators (to account for warping), default: %(default)s')
-parser.add_argument('-b','--blackseps',action="store_true",
+group_column.add_argument('-b','--blackseps',action="store_true",
                     help="also check for black column separators")
 
 # whitespace column separators
-parser.add_argument('--maxcolseps',type=int,default=3,
+group_column.add_argument('--maxcolseps',type=int,default=3,
                     help='maximum # whitespace column separators, default: %(default)s')
-parser.add_argument('--csminaspect',type=float,default=1.1,
-                    help='minimum aspect ratio for column separators')
-parser.add_argument('--csminheight',type=float,default=10,
+group_column.add_argument('--csminheight',type=float,default=10,
                     help='minimum column height (units=scale), default: %(default)s')
-
-# wait for input after everything is done
```

Comment:
This was an artefact from https://github.com/tmbdev/ocropy/commit/e39dc9d40fae913547990476f3f653994a8b7430#diff-0e349d933e2d87621d932d2ae80cf70aL64